File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/redis/CommonRedisVo.java
package org.javamaster.invocationlab.admin.model.redis;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @author yudong
 */
@EqualsAndHashCode(callSuper = true)
@Data
public class CommonRedisVo extends ValueVo {
    private String connectId;
    private Integer redisDbIndex;

    private String fieldKey;
    private Boolean fieldKeyJdkSerialize;
    private Double score;
    private String oldRedisKey;
    private String oldRedisValue;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/redis/CommonRedisVo.kt
package org.javamaster.invocationlab.admin.model.redis

import org.javamaster.invocationlab.admin.annos.AllOpen


/**
 * @author yudong
 */
@AllOpen
class CommonRedisVo : ValueVo() {
    var connectId: String? = null
    var redisDbIndex: Int? = null

    var fieldKey: String? = null
    var fieldKeyJdkSerialize: Boolean? = null
    var score: Double? = null
    var oldRedisKey: String? = null
    var oldRedisValue: String? = null
}
